crates/router/src/core/payments/operations/payment_complete_authorize.rs

```diff
@@ -110,10 +110,8 @@ impl<F: Send + Clone> GetTracker<F, PaymentData<F>, api::PaymentsRequest> for Co
 
         payment_attempt.payment_method = payment_method.or(payment_attempt.payment_method);
         payment_attempt.browser_info = browser_info;
-        payment_attempt.payment_method_type = request
```

Comment:
why are we ignoring the payment_method_type from request?

Reply:
When we try to get the `payment_method_type` from the request, it always return a `None` for recurring mandate as we'll not be using the `payment_method_type` that we get from the `get_token_pm_type_mandate_details()`.